signalfx-tracing/signalfx-dotnet-tracing/cloudfoundry-buildpack/README.md

```diff
@@ -0,0 +1,42 @@
+# CloudFoundry deployment with SignalFx Tracing Library for .NET
+
+This is an example of automatically producing traces using the
+[SignalFx Tracing Library for .NET](https://github.com/signalfx/signalfx-dotnet-tracing/).
+This example is a simple web app that just executes an HTTP call when its only endpoint is called.
+It's auto-instrumented by the SignalFx Tracing Library for .NET Buildpack.
+
+## Building the example app
+
+To run this example locally and send traces to your available Smart Agent or OpenTelemetry Collector,
+please clone this repository and from this directory do the following:
+
+```sh
+$ cake
```

Comment:
It seems to me that you could do the same without `cake`, perhaps just `dotnet publish`  with proper parameters.